collect_app/src/main/java/org/odk/collect/android/activities/DataManagerList.java

```diff
@@ -195,7 +195,7 @@ private void createDeleteInstancesDialog() {
         mAlertDialog = new AlertDialog.Builder(this).create();
         mAlertDialog.setTitle(getString(R.string.delete_file));
         mAlertDialog.setMessage(getString(R.string.delete_confirm,
-                mSelected.size()));
+                String.valueOf(mSelected.size())));
```

Comment:
It just says "Wrong argument type for formatting argument '#1' in delete_confirm: conversion is 's', received int". Another option would be to use double/int placeholders as appropriate. I decided not to go that route because all of those strings would need to be re-translated.